### ISSUE TITLE
Will ReLU be ever called for single dimensional node_state_dim?

### ISSUE THREAD

**Issue by tsjain** [simonlinyj/Graph-Matching-Networks#12]
Since the default configuration of node_hidden_sizes is a vector of length 1, will the layers in the _for_ loop on lines 38:40 ever get created?

https://github.com/Lin-Yijie/Graph-Matching-Networks/blob/f63c575ef9c6132207a52d2285cc4583c9947456/GMN/graphembeddingnetwork.py#L38

**Comment by Lin-Yijie** [comment]
No, won’t be created. If node_hidden_sizes is a vector of length 1, it will be only added on line 37.

https://github.com/Lin-Yijie/Graph-Matching-Networks/blob/f63c575ef9c6132207a52d2285cc4583c9947456/GMN/graphembeddingnetwork.py#L37
